libnetdata/ebpf/ebpf.c

```diff
@@ -88,11 +89,57 @@ int get_kernel_version() {
     while (*version) *move++ = *version++;
     *move = '\0';
 
+    fd = snprintf(out, (size_t)size, "%s.%s.%s", major, minor, patch);
+    if (fd > size)
+        error("[EBPF]: The buffer to store kernel version is not smaller than necessary.");
+
     return ((int)(str2l(major)*65536) + (int)(str2l(minor)*256) + (int)str2l(patch));
 }
 
+int get_redhat_release()
```

Comment:
This code needs to be deduplicated from the `kernel-collector/user/process_user.c` file and should change as is described in https://github.com/netdata/kernel-collector/issues/137

Reply:
I will remove it in the next commit, thank you for the suggestion!